README.md

```diff
@@ -1,13 +1,26 @@
-# Mongo API Project
+# Mongo API Project  🖥
 
-Replace this readme with your own information about your project. 
+This project follows on from my project express-api where I used a set of data locally. This time I model my database using Mongoose models, and persist the data in a database.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## View it live  👀
 
-## The problem
+https://project-mongodb.herokuapp.com/
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+## The problem  🤓
 
-## View it live
+The API I have worked with has: 
+
+- Two different routes making use of both queries and params.
+- One endpoint to return a **collection** of results (array of elements)
+- One endpoint to return a **single** result (single element).
+- The API make use of Mongoose models to model the data and use these models to fetch data from the database.
+- The API is [RESTful](https://www.smashingmagazine.com/2018/01/understanding-using-rest-api/)
+
+## Learnings  🌟
+
+During this project I have learnt:
+
+- How to model data in Mongoose
+- How to fetch items from a Mongo database using Mongoose
+- How to seed large amounts of data to a database
```

Comment:
Add the list of your endpoint, so users know how to use it ;)

Reply:
Hey Karol, thank you for your review! Not sure what you mean here - can you give an example please? Thanks!

Reply:
It's nice with some documentation so the users know what endpoints you have. Not a requirement but a real nice-to-have!